packages/components/src/ui/color-picker/rgba-color-picker.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line no-restricted-imports
+import type { ComponentProps } from 'react';
+import { RgbaColorPicker as Picker } from 'react-colorful';
```

Comment:
Nit: when renaming imports from `react-colorful`, should we actually prefix these components with `ReactColorful*` or something similar?